### ISSUE TITLE
wrong output with calc()

### ISSUE THREAD

**Issue by liuwf2012** [cssnano/cssnano#742]
![image](https://user-images.githubusercontent.com/7323242/56950434-0ff50f00-6b68-11e9-8b70-326cf90eb3c2.png)

### initial state
```css
.class {
  width: calc(-25rem - (1280px - 100vw) * 0.5);
}
```

### correct minimization
```css
.class{
  width: calc(-25rem - 640px + 50vw);
}
```

### actual minimization
```css
.class{
  width: calc(-25rem - 640px - 50vw);
}
```

**Comment by evilebottnawi** [comment]
Bug in postcss calc

**Comment by cduivis** [comment]
I just encountered the same issue:

**initial:**
`calc(0.7 * 560 * (1220px - 100vw) / 1220 + 0.1 * 560 * (1220px - 50vw) / 1220)`
**actual:**
`calc(478240px - 39200vw + 68320px - 2800vw)`

I see this bug was already reported in 2019 is some actually picking this up?

Btw for such complex behavior as redefining calculations, it would be a good thing to define some unit tests as this is easily broken.

**Comment by wszydlak** [comment]
> Bug in postcss calc

Could you provide issue number in postcss-calc? Is this already fixed?

**Comment by ludofischer** [comment]
> @alexander-akait Could you provide issue number in postcss-calc? Is this already fixed?

It's likely this one: https://github.com/postcss/postcss-calc/pull/137 There's a PR but has never been merged.

**Comment by ludofischer** [comment]
Fixed in cssnano 5.0.15. 
```css
calc(0.7 * 560 * (1220px - 100vw) / 1220 + 0.1 * 560 * (1220px - 50vw) / 1220)
```
now returns
```css
calc(448px - 34.42623vw)
```
